core/src/docker-builder/resources/lambdas/lambda.py

```diff
@@ -0,0 +1,104 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+import boto3
+import os
+import logging
+
+codebuild = boto3.client('codebuild', os.getenv('AWS_REGION'))
+log = logging.getLogger()
+log.setLevel(logging.INFO)
+
+
+def on_event(event, ctx):
+    log.info(event)
+    request_type = event['RequestType'].lower()
+    if request_type == 'create':
+        return on_create(event)
+    if request_type == 'update':
+        return on_update(event)
+    if request_type == 'delete':
+        return on_delete(event)
+    raise Exception(f'Invalid request type: {request_type}')
+
+
+def on_create(event):
+    log.info(event)
+
+    project_name = event['ResourceProperties']['codebuildProjectName']
+    s3_path = event['ResourceProperties']['s3Path']
+    ecr_uri = event['ResourceProperties']['ecrURI']
+    tag = event['ResourceProperties']['tag']
+
+    response = codebuild.start_build(
```

Comment:
We could reuse (with a bit of refacto) the cdkDeployer logic and code : https://github.com/aws-samples/aws-analytics-reference-architecture/blob/main/core/src/common/cdk-deployer.ts#L346 instead of rewriting it.

Reply:
I forgot about this method, I'll see how to reuse the code.